@here/harp-map-controls/test/MapControlsTest.ts

```diff
@@ -157,9 +159,15 @@ describe("MapControls", function () {
     before(function () {
         if (inNodeContext) {
             const theGlobal: any = global;
-            theGlobal.requestAnimationFrame = () => {};
+            theGlobal.requestAnimationFrame = (callback: (time: DOMHighResTimeStamp) => void) => {
+                setTimeout(callback, 0, 1);
+            };
+            let time = 0;
             theGlobal.performance = {
-                now: () => {}
+                now: () => {
+                    // Time in ms, i.e. 20ms gives us a FPS of 50.
+                    return (time += 20);
```

Comment:
I tried incrementing by 1, but it is just a waste of resources, so this is a good value which approaches normal usage